RecoLocalMuon/RPCRecHit/plugins/RPCPointProducer.cc

```diff
@@ -25,22 +28,55 @@
 
 // user include files
 
+void RPCPointProducer::fillDescriptions(edm::ConfigurationDescriptions& descriptions) {
+  edm::ParameterSetDescription desc;
+  desc.add<bool>("incldt", true);
+  desc.add<bool>("inclcsc", true);
+  desc.add<bool>("incltrack", false);
+  desc.add<bool>("debug", false);
+  desc.add<double>("rangestrips", 4.);
+  desc.add<double>("rangestripsRB4", 4.);
+  desc.add<double>("MinCosAng", 0.85);
+  desc.add<double>("MaxD", 80.0);
+  desc.add<double>("MaxDrb4", 150.0);
+  desc.add<double>("ExtrapolatedRegion", 0.6);
```

Comment:
We are currently using the parameters of the RecoLocalMuon/RPCRecHit/test/testRPCPointProducer.py

Reply:
> We are currently using the parameters of the RecoLocalMuon/RPCRecHit/test/testRPCPointProducer.py

Indeed they should be reverted to the previous default, otherwise the other tests which link them would get affected, see https://cmssdt.cern.ch/lxr/search?%21v=CMSSW_12_5_X_2022-08-18-2300&_filestring=&_string=rpcPointProducer_cff&_casesensitive=1
The test parameters are overwritten anyhow in your testRPCPointProducer.py

Reply:
There are some different parameters in RecoLocalMuon/RPCRecHit/plugins/RPCPointProducer.cc required by the segment extrapolation method, so they were added. Also for the old parameters in RecoLocalMuon/RPCRecHit/python/rpcPointProducer_cfi.py that may be used by other tests, we can add them as additional parameters in RPCPointProducer.cc to comply with the old ones.